release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -1548,6 +1548,11 @@ features marked *GA* indicate _General Availability_.
 |-
 |TP
 
+|Multi-stage builds in Dockerfiles managed by the S2I image builder
```

Comment:
I suspect ‘S2I image builder’ here if correct, most display 'image builder'.
See https://github.com/openshift/imagebuilder#oci-image-builder 
S2I means source-to-image.